workers/unity/Assets/Playground/Scripts/MonoBehaviours/DeleteCubeCommandReceiver.cs

```diff
@@ -71,7 +71,7 @@ private void OnDeleteEntityResponse(WorldCommands.DeleteEntity.ReceivedResponse
             }
 
             var spawnedCubesCopy =
-                new List<EntityId>(CubeSpawnerInputBehaviour.GetSpawnedCubes(cubeSpawnerWriter.Data));
+                new List<EntityId>(cubeSpawnerWriter.Data.SpawnedCubes);
```

Comment:
We should just remove this. I keep commenting on this code that removing it from that list is wrong.

Reply:
Maybe so - but out of scope for this PR imp

Reply:
Its large enough already 😬

Reply:
Fair enough